synfig-core/src/synfig/filesystemnative.h

```diff
@@ -31,6 +31,12 @@
 /* === H E A D E R S ======================================================= */
 
 #include "filesystem.h"
+#include "general.h" // synfig::error(...)
+
+#ifdef _WIN32
+#include <codecvt>
```

Comment:
Sadly codecvt is deprecated in C++17. But let's use while we can :)

Reply:
Thanks for pointing this out.
In this case, we can replace it with `MultiByteToWideChar` function.

Reply:
> Sadly codecvt is deprecated in C++17. But let's use while we can :)

I think we can use it until C++17, after switch to C++17 we don't need it because we will have `std::filesystem` class.